README.md

```diff
@@ -20,10 +22,10 @@ clickStream.observe(event => console.log(event));
 
 #### &lt;eventName&gt; :: (EventTarget t, Event e) => t &rarr; boolean=false &rarr; Stream e
 
-See [the source](src/index.js) for all the supported event names.  Each has the general signature:
+See [the source](src/index.js) for all the supported event names. Each has the general signature:
 
 ```js
-const stream = eventName(domNode, useCapture = false)
+const stream = eventName(domNode, (useCapture = false));
```

Comment:
Hmm, I think this was just a confusing way to try to document these functions to start with.  How about this: Undo the addition of the parens to this line, and the similar `domEvent` line below, and I'll change how we documented this to a style that's similar to the `@most/core` docs.  Sound ok?

Reply:
Sure :+1: